runbook.md

```diff
@@ -0,0 +1,17 @@
+# Runbook for proposed changes with steps
+
+## -- changes:
+ 1. secure access to prod trust claim DB
+ 2. change digital ocean droplet to test server with a separate DB for trust claim
+ 3. maintain digital ocean for Taiga and Aws for trust claim
+ 4. set-up CI/CD on dev (digital ocean trustclaim server)
+ 5. rewrite taiga.whatscookin.us to redirect to Taiga.linkedtrust.us
+    
+## -- steps:
+ 1. for secure access to prod DB, I'll only allow the Aws prod IP to be the only inbound traffic IP
+2. for this, I'll enable the DNS name 'lives.linkedtrust.us' to be the dev domain and the IP being the dev IP, which is on CI/CD with the master branch on both both front and backend repo, also I'll need the digital ocean webhook to be attached to both front and backend repo for me to enable it with Jenkins.
+3. I'll create a snapshot of the Taiga EC2 in Aws, and turn it down, since all traffic is on the digital ocean droplet, also should I move Taiga dB to a separate DB or leave it as it's usually the local one available in the same server as the program codes (local pg dB)????
+4. for this, I'll need the webhook for the digital ocean Jenkins to be included to both the front and backend repo
```

Comment:
yes correct!  do you still not have maintain access?  I will do it! 

*** Also *** we need trust-claim-data-pipeline as well, that also needs to run in crontab, this should be checked out on dev server automatically as well!  It could later run on separate server or as a lambda type thing, but for now it seems ok to just run it all on the one server.

Reply:
sure ma'am, so should i run it?